.github/workflows/offline-docs.yml

```diff
@@ -0,0 +1,40 @@
+name: Update Offline Docs
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.7.2'
+
+      - name: Build docs
+        run: |
+          git clone --depth=1 --recursive https://github.com/theos/theos.dev tmp
+          cd tmp
+          make all
```

Comment:
consolidate with `make -C tmp`

also fixes what appears to be a bug below: the current working directory is `tmp` and then we `rm -rf tmp`

Reply:
Thanks for catching all of these!